apps/frontend/app/(client)/_libs/queries/submission.ts

```diff
@@ -0,0 +1,36 @@
+import { queryOptions } from '@tanstack/react-query'
+import {
+  getSubmissions,
+  getSubmissionDetail,
+  type GetSubmissionsRequest,
+  type GetSubmissionDetailRequest
+} from '../apis/submission'
+
+export const submissionQueries = {
+  all: (contestId: number) => ['contest', contestId, 'submission'] as const,
```

Comment:
```suggestion
  all: ({contestId, problemId}: {contestId: number; problemId: number; }) => ['submission', 'contest', contestId, { problemId }] as const,
```

all 키에 `problemId`를 포함하는 건 어떨까요?? submission 데이터 가져올 때 필요한 필수 파라미터들은 all 키에 포함하면 좋을 것 같아요!

이거에 맞춰서 다른 키들도 수정 부탁드려요~

Reply:
코멘트들 수정해봤습니다!!  >_<... 꼼꼼하게 봐주셔서 감사합니다!!!!